x/emissions/keeper/whitelist.go

```diff
@@ -0,0 +1,321 @@
+package keeper
+
+import (
+	"context"
+
+	"cosmossdk.io/collections"
+	errorsmod "cosmossdk.io/errors"
+
+	"github.com/allora-network/allora-chain/x/emissions/types"
+)
+
+/// SETTERS - Functions that update whitelists
+
+func (k *Keeper) AddWhitelistAdmin(ctx context.Context, admin ActorId) error {
+	if err := types.ValidateBech32(admin); err != nil {
+		return errorsmod.Wrap(err, "error validating admin id")
+	}
+	return k.whitelistAdmins.Set(ctx, admin)
+}
+
+func (k *Keeper) RemoveWhitelistAdmin(ctx context.Context, admin ActorId) error {
+	if err := types.ValidateBech32(admin); err != nil {
+		return errorsmod.Wrap(err, "error validating admin id")
+	}
+	has, err := k.whitelistAdmins.Has(ctx, admin)
+	if err != nil {
+		return err
+	} else if !has {
+		return nil
+	}
+	return k.whitelistAdmins.Remove(ctx, admin)
```

Comment:
```suggestion
	return k.whitelistAdmins.Remove(ctx, admin)
```
The call to `KeySet.Remove` is already a no-op if the key is not present, no need to check the its existence before, we can reduce state access 😸

(This is also present in other places)

Reply:
great catch! updated everything in `whitelist.go` with this in mind